src/importexport/musicxml/tests/data/testSystemBrackets3_ref.mscx

```diff
@@ -32,56 +32,11 @@
         </Staff>
       <trackName>Backing
 Vocals</trackName>
-      <Instrument id="voice">
+      <Instrument>
         <longName>Backing
 Vocals</longName>
         <shortName>B. Vx.</shortName>
         <trackName>Voice</trackName>
-        <minPitchP>38</minPitchP>
```

Comment:
I don't know why there are so many changes to this file, but it looks correct when loaded...note that prior to the additional of testSystemBrackets3 it wasn't being used as part of the MU4 tests.